extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -100,51 +102,73 @@ static public function canShowDiscussions( $cityId, $ignoreWgEnableRecirculation
 	}
 
 	private static function addLiftIgniterMetadata( OutputPage $outputPage ) {
-		global $wgCityId,
-		       $wgDevelEnvironment,
-		       $wgEnableArticleFeaturedVideo,
-		       $wgIsPrivateWiki,
-		       $wgLanguageCode,
-		       $wgStagingEnvironment,
-		       $wgWikiaEnvironment;
+		$metaData = self::getMetaData();
+		$metaDataJson = json_encode( $metaData );
+
+		$outputPage->addScript(
+			"<script id=\"liftigniter-metadata\" type=\"application/json\">${metaDataJson}</script>"
+		);
+	}
 
+	private static function getMetaData() {
+		global $wgLanguageCode, $wgCityId, $wgEnableArticleFeaturedVideo;
 		$context = RequestContext::getMain();
 		$title = $context->getTitle();
 		$articleId = $title->getArticleID();
-		$metaData = [];
+		$shouldNoIndex = self::shouldIndex();
```

Comment:
personally i'm a little confused with `$shouldNoIndex` and `shouldIndex()`; so maybe consider changing func name to `should(Have)NoIndex` or sth similar?